targets/UnrealMarketplacePlugin/examplesource/ExampleProject/ExampleProject.uproject.ejs

```diff
@@ -9,5 +9,13 @@
             "Type": "Runtime",
             "LoadingPhase": "Default"
         }
-    ]
+    ],
+	"TargetPlatforms": [
+		"Android",
+		"AllDesktop",
+		"IOS",
+		"MacNoEditor",
+		"WindowsNoEditor",
+		"LinuxNoEditor"
```

Comment:
>		 [](start = 0, length = 2)

Let's replace these tabs with spaces for consistency with the rest of code

Reply:
/Agree